tests/pipelines/test_pipelines_common.py

```diff
@@ -696,7 +697,9 @@ def test_cpu_offload_forward_pass(self, expected_max_diff=1e-4):
         inputs = self.get_dummy_inputs(torch_device)
         output_with_offload = pipe(**inputs)[0]
 
-        max_diff = np.abs(to_np(output_with_offload) - to_np(output_without_offload)).max()
+        max_diff = numpy_cosine_similarity_distance(
```

Comment:
https://github.com/huggingface/diffusers/pull/4812/files#r1307509747 would be better